bedrock/firefox/views.py

```diff
@@ -494,26 +489,14 @@ def new(request):
     # if no/incorrect scene specified, show scene 1
     else:
         if lang_file_is_active('firefox/new/wait-face', locale) and experience == 'waitface':
-            if switch('experiment-firefox-new-waitface'):
-                if variant == 'b':
-                    template = 'firefox/new/wait-face/scene1-video.html'
-                else:
-                    template = 'firefox/new/wait-face/scene1.html'
-            elif switch('experiment-firefox-new-waitface-switch'):
-                if variant == '2':
-                    template = 'firefox/new/wait-face/scene1-newcopy.html'
-                elif variant == '3':
-                    template = 'firefox/new/wait-face/scene1-switch.html'
-                else:
-                    template = 'firefox/new/wait-face/scene1.html'
-            else:
-                template = 'firefox/new/wait-face/scene1.html'
+            template = 'firefox/new/wait-face/scene1.html'
         elif lang_file_is_active('firefox/new/reggiewatts', locale) and experience == 'reggiewatts':
             template = 'firefox/new/reggie-watts/scene1.html'
         else:
             template = 'firefox/new/scene1.html'
 
-    return l10n_utils.render(request, template, {'v': variant})
+    # sending empty context so we don't have to update the tests every time we need a variant value
+    return l10n_utils.render(request, template, {})
```

Comment:
@pmac - Is this horrible? Any other suggestions here? It seems silly to have to re-add `ANY` to all tests of this view when we need a variation sent in `context`.

Reply:
Well, I see you no longer need variant in the context, but you also don't need to not have it... if you expect to need variant again then I see no compelling reason for you to remove it. You can just not delete the bits from lines 479-485, and maybe add a comment. I think that'd be less confusing.